packages/subapp-web/test/spec/util.spec.js

```diff
@@ -19,3 +22,27 @@ describe("loadAssetsFromStats", () => {
     expect(assets).be.empty;
   });
 });
+
+describe("getChunksById", function() {
+  const prodStats = JSON.parse(Fs.readFileSync(Path.join(__dirname, "../data/prod-stats.json")));
+  const devStats = JSON.parse(Fs.readFileSync(Path.join(__dirname, "../data/dev-stats.json")));
+
+  it("should handle production stats", () => {
+    const result = getChunksById(prodStats);
+
+    expect(result).to.have.property("_names_");
```

Comment:
If you expect the list of properties to grow, it might be more readable to switch to "to.have.all.keys"